src/webhook/test_namespace_validation.py

```diff
@@ -4,52 +4,6 @@
 from webhook import namespace_validation
 
 
-PRIVILEGED_NAMESPACES = (
```

Comment:
Why are the namespace tests changing in a PR about group validation tests? Will these changes introduce merge conflicts with https://github.com/openshift/managed-cluster-validating-webhooks/pull/66 (cc @jewzaam)?